src/components/App/filters/filters.js

```diff
@@ -1,7 +1,10 @@
 import { i18n } from '@/i18n'
 import moment from 'moment'
 import store from '../../../store'
-
+import { base } from '@tokend/js-sdk'
+import { CREATE_ASSET_REQUEST_STATES, LIMITS_REQUEST_STATES_STR, STELLAR_TYPES } from '@/constants'
+import { ASSET_REQUEST_TYPES } from '@/components/User/Assets/AssetRequests/components/AssetRequestList'
```

Comment:
fix it. Do not import const from component file

Reply:
done